### DISCUSSION TITLE
Direction to port/create a custom theme.

### DISCUSSION THREAD

**Discussion by zoedsoupe** [helix-editor/helix#8553] [Q&A]
Community has already done a awesome work on porting/creating themes to helix. Then I would like to contribute too porting some of my favorites neovim themes like [melange](https://github.com/savq/melange-nvim) or [komau](https://github.com/ntk148v/komau.vim) . But I’m very newbie into theme creation thing so would you give me any initial directions? Based on a existing theme, how could I port it to helix definition?

**Comment by nshern** [comment]
You can start by taking a look at the [user-submitted themes](https://github.com/helix-editor/helix/tree/master/runtime/themes) and trying to reverse-engineer them?
And [here](https://github.com/n0s4/helix-theme-template/blob/main/template.toml) is a template file to create your own theme.
